mrp_mto_with_stock/models/mrp_production.py

```diff
@@ -99,6 +99,13 @@ def _prepare_mto_procurement(self, move, qty, mto_with_no_move_dest_id):
             vals['move_dest_id'] = move.id
         return vals
 
+    # This method should be overriden in submodule to manage cases where
+    # we need to add quantities to the forecast quantity. Like draft purchase
+    # order, purchase request, etc...
+    @api.model
+    def _get_incoming_qty_waiting_validation(self, move):
+        return 0.0
```

Comment:
Put the comments of the method in a triple quote comment after the method declaration
```python
def _get_incoming_qty_waiting_validation(self, move):
    """comment"""
    return
```